observability/monitoring/configuring-core-platform-monitoring/configuring-alerts-and-notifications.adoc

```diff
@@ -57,13 +57,13 @@ include::modules/monitoring-configuring-alert-routing-default-platform-alerts.ad
 [role="_additional-resources"]
 .Additional resources
 
-* link:https://access.redhat.com/solutions/6828481[Send test alerts to Alertmanager in OpenShift 4] (Red{nbsp}Hat Customer Portal)
+* link:https://access.redhat.com/solutions/6828481[Send test alerts to Alertmanager in OpenShift 4]
```

Comment:
Is the link to this Solution article reliable? I think the parentheses would be useful here as well as a log on might be needed.